src/OpenApi/test/Integration/snapshots/OpenApiDocumentIntegrationTests.VerifyOpenApiDocument_documentName=responses.verified.txt

```diff
@@ -186,6 +176,22 @@
             "format": "date-time"
           }
         }
+      },
+      "Triangle": {
```

Comment:
We now have ShapeTriangle and Triangle, is that expected?

Reply:
Yep, this is expected since they represent two different schemas. `Triangle` is the schema we create when the `Triangle` type is consumed directly. `ShapeTriangle` is part of the polymorphic schema for `Shape`. The difference between the two is that `ShapeTriangle` requires that a discriminator property be defined whereas `Triangle` doesn't.